packages/quantic/force-app/main/default/lwc/quanticSearchBoxSuggestionsList/quanticSearchBoxSuggestionsList.js

```diff
@@ -89,6 +89,10 @@ export default class QuanticSearchBoxSuggestionsList extends LightningElement {
     this.dispatchEvent(suggestionSelectedEvent);
   }
 
+  handleSuggestionHover() {
+    this.resetSelection();
```

Comment:
Interesting this, what was the behavior exactly?
Assuming something funky whenever you'd hover a suggestion but press the up/down arrow after?

Reply:
if you were keyboard navigating then brought the mouse cursor onto the list, the item highlighted by the keyboard would remain highlighted, in addition to the mouse highlight.

Reply:
if you're mouse hovering over the list then leave it there as you start using the keyboard, then the keyboard highlighting won't work until you move the mouse away. I think that makes more sense. What do you think?